config/services.yaml

```diff
@@ -4,6 +4,11 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
+    app.s3_client.version: '2018-10-16'
+    app.s3_client.region: 'ile-de-france'
```

Comment:
This config should not be hardcoded (this will change based on your environment)